src/pykeen/pipeline.py

```diff
@@ -834,9 +834,11 @@ def pipeline(  # noqa: C901
     if not metadata:
         metadata = {}
     title = metadata.get('title')
+    tags = metadata.get('run_tags', None)
 
     # Start tracking
     result_tracker.start_run(run_name=title)
+    result_tracker.set_tags(tags)
```

Comment:
Since it's up to us as the developers to implement both `start_run()` and `set_tags()`, I don't see much benefit to them being separate functions (they're never called in any other way - the ResultTracker subclasses are an example of the "adapter" pattern if you want to get all high brow and software engineery)

Instead, I think it would make the most sense just to add additional code to the MLFlowResultTracker's implementation of `start_run()` to also set the tags, which should be available as instance variables (see comment above)

Reply:
yes that would work fine I think - I will update it. Thank you

Reply:
Okay - I updated it.. not sure if you think its the right way or not.. Also thanks for Homebrew.. was very smooth and thanks for class diagram.. I am still learning pykeen, mostly cause I like the pipelines hence, its very helpful.